release.md

```diff
@@ -1,13 +1,16 @@
 
-# Netmaker v0.20.0
+# Netmaker v0.20.1
 
 ## whats new
-- New UI
-- revamped compose-files and install scripts
-- TURN
+- 
     
 ## whats fixed
-- Caddy does not handle netmaker exporter well for EE
+- enrollment keys for non-admins 
+- client version displayed correctly in UI
+- upd hole punching improvments
+- SSL fallback to letsencrypt
+- permission handling for non-admin users
+
```

Comment:
Additional fixes:
- Client QR code and downloadable config file name
- Improved stability in Netmaker UI (null host/node errors)

Reply:
Those items are more applicable to netmaker-ui release notes are they not?

Reply:
Yes, UI. Will we start having release notes for the UI?

Reply:
manual for now as there is no ui release workflow yet